visualization/app/codeCharta/ui/filePanel/filePanel.component.html

```diff
@@ -78,4 +78,10 @@
 			</md-option>
 		</md-select>
 	</md-input-container>
+
+	<div class="file-button global-settings">
```

Comment:
The global settings are not part of a file. We should create a Container-Component around the file-panel AND the global settings button. In that container you can arrange the two components next to each other.

Reply:
I Added a toolbar component which is basically the top bar and it contains the 
- import button
- download button
- filePanel component switch and selects
- global settings button